dub.selections.json

```diff
@@ -1,7 +1,9 @@
 {
 	"fileVersion": 1,
 	"versions": {
-		"argparse": "1.2.0",
-		"fswatch": "0.6.0"
+		"argparse": "~master",
+		"fswatch": "0.6.0",
+		"toml": "2.0.1",
+		"unit-threaded": "2.0.5"
```

Comment:
CC @WebFreak001 this shouldn't be there. I can make a PR if I have time.

Reply:
Humm, the dub config on toml seems correct :thinking: . Maybe its a behavior of dub, but I suppose this dependency doesn't affect the outer dependency, so no need to be version pinned, because it's only used internally. Perhaps have a dedicated `internal-unittest` configuration on dub config?